source/getting-started/autostart-systemd.markdown

```diff
@@ -63,6 +63,13 @@ $ sudo systemctl status home-assistant@[your user] -l
 [...]
 ```
 
+IF, you get an error message, and it fails to start Home Assistant, double check that your ExecStart=/usr/bin/hass points to the correct location...it's likely ExecStart=/usr/local/bin/hass  If you didn't update it correctly, you may have to remove the home-assistant@pi.service file and then start over at the top of this instruction set using the correct directory.
```

Comment:
Please follow the same style here and embed stuff like `ExecStart=` with \`. It would also be possible to simple move and edit the file. 

`...it's likely ExecStart=/usr/local/bin/hass  If you` looks a bit misplaced.